### PR TITLE
tracee-ebpf: simplify saving to buf

### PR DESCRIPTION
**Pull request by yanivagman**

### CODE REVIEW

File: tracee-ebpf/tracee/tracee.bpf.c [hunk 1]
```diff
@@ -1462,6 +1462,11 @@ static __always_inline int save_path_to_str_buf(buf_t *string_p, const struct pa
     unsigned int off;
     int sz;
 
+    // Get per-cpu string buffer
```
Comment (grantseltzer) [inline tracee-ebpf/tracee/tracee.bpf.c:1466]: BPF programs can't be preempted and moved to a different CPU, right?

**Review by grantseltzer — COMMENTED** [review]
Do I understand correctly that you're calculating the index of the string buffer when you originally create it, and then use it when accessing the buffer, instead of recalculating on each access?

File: tracee-ebpf/tracee/tracee.bpf.c (hunk 1 above)
Comment (yanivagman) [reply tracee-ebpf/tracee/tracee.bpf.c:1466]: Right

**Review by grantseltzer — APPROVED** [review]
LGTM!

**Pull request merged**